### DISCUSSION TITLE
How to add GetGrass.io app?

### DISCUSSION THREAD

**Discussion by jokinvipo** [engageub/InternetIncome#116] [Q&A]
Hey,

i was trying to add getgrass to this project, any tip how can i run it?

thanks alot

**Comment by engageub** [comment]
Please use test branch for grass.

Thank you

**Reply by engageub** [reply]
Please find the link if you are not able to figure that out.

https://github.com/engageub/InternetIncome/tree/test

Thank you

**Reply by jokinvipo** [reply]
Hello, 

Its working smoothly right now. Is good to use this test version for all apps instead the stable one?

Thanks alot for the help

**Reply by engageub** [reply]
Yes, test branch is for advanced users. It has multiple network configurations and also docker images which are not yet available in main branch.
You can use the test branch.

Thank you

**Comment by engageub** [comment]
Just to let you know grass has been pushed to main branch. 

https://github.com/engageub/InternetIncome/tree/main

Thank you
